src/Cake.Common/Tools/NuGet/Pack/NuSpecContent.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Cake.Common.Tools.NuGet.Pack
+{
+    /// <summary>
+    /// Represents a NuGet nuspec file
+    /// </summary>
+    public class NuSpecContent
+    {
+        /// <summary>
+        /// The location of the file or files to include. The path is relative to the NuSpec file unless an absolute path is specified. The wildcard character, *, is allowed. Using a double wildcard, **, implies a recursive directory search.
+        /// </summary>
+        public string Src { get; set; }
```

Comment:
Should we call this one `Source` instead of `Src`, or should it map 1:1 with the nuspec name?